language-puppet.cabal

```diff
@@ -2,7 +2,7 @@
 -- documentation, see http://haskell.org/cabal/users-guide/
 
 name:                language-puppet
-version:             1.3.13
+version:             1.3.13.33
```

Comment:
Why the 33?

Reply:
My mistake. Sorry. `nix` is playing some tricks on me sometimes and this is to ensure I am tested the latest local version (not a caching one) ;-)